### ISSUE TITLE
Data transform when training

### ISSUE THREAD

**Issue by tao-sun** [fangwei123456/Parametric-Leaky-Integrate-and-Fire-Spiking-Neuron#9]
Can you please help me understand the transform added to some dataset when training? E.g. 

if dataset_name == 'MNIST':
        transform_train = transforms.Compose([
            transforms.RandomAffine(degrees=30, translate=(0.15, 0.15), scale=(0.85, 1.11)),
            transforms.ToTensor(),
            transforms.Normalize(0.1307, 0.3081),
        ])

I tried to train without the RandomAffine and it seemed that the performance was not that good as with this transform.

**Comment by fangwei123456** [comment]
Hi, thest transforms are used to reduce overfitting. If you do not use them, your network are easily to overfit.

Refer to this for more details:

https://pytorch.org/vision/main/transforms.html

**Comment by tao-sun** [comment]
Thanks for your quick response.

How do you choose the parameters in the transforms (i.e. degrees=30, translate=(0.15, 0.15), scale=(0.85, 1.11))?

**Comment by fangwei123456** [comment]
> How do you choose the parameters in the transforms

It is set by comparing experiment results with different hyper-parameters.
